package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/ImportHelpers.cs

```diff
@@ -59,7 +59,7 @@ public static GameObject ImportSceneAsGameObject(Scene scene, GameObject parent
         }
 
 #if UNITY_EDITOR
-        public static string ImportAsPrefab(Scene scene, string prefabPath = null)
+        public static string ImportAsPrefab(Scene scene, SceneImportOptions importOptions, string prefabPath = null)
```

Comment:
I don't think we should be changing public API in a test PR. afaict, it's not required for the tests, so I think we should consider doing this in a separate PR, if at all

Reply:
The functionality to set the SceneImportOptions on Import is used for the testcase: ImportAsPrefab_TextureDataImported

But yes, I do agree that changing a public API in a test PR isnt very ideal
I will create a separate PR to change this part

But I would also like to know, if technical design wise, this is a change that other devs can agree to (allowing custom SceneImportOptions for ImportAsPrefab function)

Reply:
Aha - my search for `ImportAsPrefab(` did not find it :-) .

I think we need to do a pass of the API at some point and make these kinds of decisions, and we should add documentation etc as we go. imo this PR should add tests that validate the API/package in its current form, and we can modify/add tests later if and when we change the API.

Reply:
This sounds good for me!

I will revert these changes, and for now, the testcase implementation as well - I've added a TODO tag for the case to come back to it later!